snuba/query/dsl_mapper.py

```diff
@@ -33,8 +31,7 @@
 
 def and_cond_repr(exp: Expression, visitor: ExpressionVisitor[str]) -> str:
     assert isinstance(exp, FunctionCall)
-    conditions = get_first_level_and_conditions(exp)
-    parameters = ", ".join([arg.accept(visitor) for arg in conditions])
+    parameters = ", ".join([arg.accept(visitor) for arg in exp.parameters])
```

Comment:
the way mapper was creating conditions did not reflect the structure of the query object, bc it was rebalancing